### PR TITLE
W-14235997-OAS-Flight-example-fa

### PR DESCRIPTION
**Pull request by failup**

### CODE REVIEW

File: modules/ROOT/pages/tut-af-design-am-flights-api.adoc [hunk 1]
```diff
@@ -67,27 +67,74 @@ To open the panel:
 .. Select *Mule DX Server* from the drop-down menu in the panel.
 +
 For more guidance, see xref:configure-default-output-panel.adoc[].
-. In `american-flights-api.raml`, paste this RAML code for the `/flights` endpoint and `get` method into the file:
+. In the `american-flights-api` file, paste this  code for the `/flights` endpoint and `get` method:
```
Comment (dukesphere) [inline modules/ROOT/pages/tut-af-design-am-flights-api.adoc]:
```suggestion
. In the `american-flights-api` file, paste this code for the `/flights` endpoint and `get` method:
```

File: modules/ROOT/pages/tut-af-design-am-flights-api.adoc [hunk 2]
```diff
@@ -67,27 +67,74 @@ To open the panel:
...
+====
+OAS Example::
+
+[source,YAML]
+--
+openapi: 3.0.0
+info:
+  title: American Flights API
+
+paths:
```
Comment (dukesphere) [inline modules/ROOT/pages/tut-af-design-am-flights-api.adoc:111]: This /{ID} endpoint needs to get updated.

Comment (dukesphere) [reply modules/ROOT/pages/tut-af-design-am-flights-api.adoc:111]: See the change to the RAML: https://dev-docs-internal.kqa.msap.io/beta-code-builder/anypoint-code-builder/tut-af-design-api-add-request-response-details (built from branch open-beta-ga)

File: modules/ROOT/pages/tut-af-design-am-flights-api.adoc [hunk 3]
```diff
@@ -42,19 +42,19 @@ If you receive the error *Mule DX API Component was not installed*, wait for Mul
 | *Project Name* | *American Flights API* +
                     Alternatively, provide a similar name of your choice. The name must be unique. The name of the project and specification file is based on the name you provide.
 | *Project Location* | Your home directory or another directory you create (see xref:start-add-folders.adoc[])
-| *API Specification Language* | *RAML 1.0* +
+| *API Specification Language* | *RAML 1.0* - *OAS 3.0* +
```
Comment (dukesphere) [inline modules/ROOT/pages/tut-af-design-am-flights-api.adoc]:
```suggestion
| *API Specification Language* | *RAML 1.0* or *OAS 3.0* +
```

File: modules/ROOT/pages/tut-af-design-am-flights-api.adoc [hunk 4]
```diff
@@ -42,19 +42,19 @@ If you receive the error *Mule DX API Component was not installed*, wait for Mul
...
                                   Anypoint Code Builder supports OAS and RAML. 
 | *Business Group* | Group in Anypoint Platform for your API specification (see xref:access-management::business-groups.adoc[])
 |===
-. Select *Create Project* to generate the American Flights API project file: `american-flights-api.raml`.
+. Select *Create Project* to generate the American Flights API project file: `american-flights-api`.
 +
```
Comment (dukesphere) [inline modules/ROOT/pages/tut-af-design-am-flights-api.adoc:50]: shoul

File: modules/ROOT/pages/tut-af-design-am-flights-api.adoc [hunk 5]
```diff
@@ -42,19 +42,19 @@ If you receive the error *Mule DX API Component was not installed*, wait for Mul
...
 The file name is based on the name you provide for the project.
```
Comment (dukesphere) [inline modules/ROOT/pages/tut-af-design-am-flights-api.adoc]:
```suggestion
The file name is based on the name you provide for the project.
```

Should we say that the file name and extension are based on the name that you provide for the project and the API specification language you use.

File: modules/ROOT/pages/tut-af-design-am-flights-api.adoc [hunk 6]
```diff
@@ -67,27 +67,74 @@ To open the panel:
...
+--
+openapi: 3.0.0
+info:
+  title: American Flights API
+
+paths:
+  '/{ID}':
+    get:
+      responses:
+        '200':
```
Comment (dukesphere) [inline modules/ROOT/pages/tut-af-design-am-flights-api.adoc]:
```suggestion
paths:
  '/{ID}':
    get:
```

Comment (dukesphere) [reply modules/ROOT/pages/tut-af-design-am-flights-api.adoc]: Not needed on this page. It's not mentioned until the next page in the tutorial.

File: modules/ROOT/pages/tut-af-design-am-flights-api.adoc [hunk 7]
```diff
@@ -67,27 +67,74 @@ To open the panel:
...
+RAML Example::
+[source,RAML]
 --
 #%RAML 1.0
 title: American Flights API
 
 /flights:
   get:
+    responses:
+      200:
```
Comment (dukesphere) [inline modules/ROOT/pages/tut-af-design-am-flights-api.adoc]:
```suggestion
```
Same comment. not documented on this page. Next page.

File: modules/ROOT/pages/tut-af-design-am-flights-api.adoc [hunk 8]
```diff
@@ -67,27 +67,74 @@ To open the panel:
...
+
+[source,YAML]
+--
+openapi: 3.0.0
+info:
+  title: American Flights API
+
+paths:
+  '/{ID}':
+    get:
```
Comment (dukesphere) [inline modules/ROOT/pages/tut-af-design-am-flights-api.adoc:117]: Same comment: It wasn't documented on this page, so let's not document the responses part here.

File: modules/ROOT/pages/tut-af-design-am-flights-api.adoc [hunk 9]
```diff
@@ -67,27 +67,74 @@ To open the panel:
...
-----
+--
 #%RAML 1.0
 title: American Flights API
 
 /flights:
   get:
-  post:
-----
+    responses:
```
Comment (dukesphere) [inline modules/ROOT/pages/tut-af-design-am-flights-api.adoc]: Same

Comment (dukesphere) [reply modules/ROOT/pages/tut-af-design-am-flights-api.adoc]: Why is post: removed?

File: modules/ROOT/pages/tut-af-design-am-flights-api.adoc [hunk 10]
```diff
@@ -67,27 +67,65 @@ To open the panel:
...
+[source,YAML]
+--
+openapi: 3.0.0
+info:
+  title: American Flights API
+
+paths:
+  '/{ID}':
+    get:
+      post: {}
```
Comment (lbauret) [inline modules/ROOT/pages/tut-af-design-am-flights-api.adoc:114]: Post endpoint should be at the same level as get, both with:
```
get: {}
post: {}
```

File: modules/ROOT/pages/tut-af-design-am-flights-api.adoc [hunk 11]
```diff
@@ -67,27 +67,65 @@ To open the panel:
...
-----
+--
 #%RAML 1.0
 title: American Flights API
 
 /flights:
   get:
-  post:
-----
+    post:
```
Comment (lbauret) [inline modules/ROOT/pages/tut-af-design-am-flights-api.adoc:125]: Post should at same level as get

File: modules/ROOT/pages/tut-af-design-api-add-request-response-details.adoc [hunk 12]
```diff
@@ -64,20 +64,66 @@ The IDE adds the fragment as a dependency to *Project Dependencies* and  `exchan
...
+        '200':
+          description: OK
+          content:
+            application/json: //<2>
+              schema:
+                type: AmericanFlight[]
+              Examples: //<3>
+                  $ref: '#/components/examples/AmericanFlightsExample'
+
+	post:
```
Comment (lbauret) [inline modules/ROOT/pages/tut-af-design-api-add-request-response-details.adoc:116]: Post should be at same level as gett

File: modules/ROOT/pages/tut-af-design-api-add-request-response-details.adoc [hunk 13]
```diff
@@ -148,9 +221,59 @@ value:
...
+              - CLE
+      responses://<1>
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: AmericanFlight[]
+
+	post:
```
Comment (lbauret) [inline modules/ROOT/pages/tut-af-design-api-add-request-response-details.adoc:260]: Post shoul be at same level as get

File: modules/ROOT/pages/tut-af-design-api-add-request-response-details.adoc [hunk 14]
```diff
@@ -198,7 +322,56 @@ Define an example for the `post` request.
...
+              - CLE
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: AmericanFlight[]
+
+	post:
```
Comment (lbauret) [inline modules/ROOT/pages/tut-af-design-api-add-request-response-details.adoc:355]: Post should be at same level as get

File: modules/ROOT/pages/tut-af-design-api-add-request-response-details.adoc [hunk 15]
```diff
@@ -198,7 +322,56 @@ Define an example for the `post` request.
...
+    body: //<1>
```
Comment (lbauret) [inline modules/ROOT/pages/tut-af-design-api-add-request-response-details.adoc:356]: Body should be inside the post node

File: modules/ROOT/pages/tut-af-design-api-add-request-response-details.adoc [hunk 16]
```diff
@@ -220,10 +393,13 @@ types:
...
-    body: //<1>
-      application/json:
-        type: AmericanFlight //<2>
+    post:
+    requestBody://<1>
+      content:
+        application/json:
+          schema: 
+          type: AmericanFlight//<2>
+
```
Comment (lbauret) [inline modules/ROOT/pages/tut-af-design-api-add-request-response-details.adoc:402]: No need to change this, it was correct as it was formerly

File: modules/ROOT/pages/tut-af-design-api-add-request-response-details.adoc [hunk 17]
```diff
@@ -198,7 +322,56 @@ Define an example for the `post` request.
...
+              schema:
+                type: AmericanFlight[]
+
+	post:
+    body: //<1>
+      application/json:
+        type: AmericanFlight //<2>
+
+'/{ID}':
+ get:
```
Comment (lbauret) [inline modules/ROOT/pages/tut-af-design-api-add-request-response-details.adoc:361]: get should be inside the /{ID} endpoint

File: modules/ROOT/pages/tut-af-design-api-add-request-response-details.adoc [hunk 18]
```diff
@@ -254,13 +455,86 @@ value:
...
 +
 To simulate a new flight record, this example does not provide an `ID` parameter.
-. In the `american-flights-api.raml` file, add the example as a response to the `post` method, and include a custom message.
+. In the `american-flights-api` file, add the example as a response to the `post` method, and include a custom message.
 +
 .Notice the `!include` directive at number 5:
+[tabs]
+====
+
+OAS Example::
```
Comment (lbauret) [inline modules/ROOT/pages/tut-af-design-api-add-request-response-details.adoc:467]: Apply same fixes as described above

File: modules/ROOT/pages/tut-af-design-am-flights-api.adoc (hunk 11 above)
Comment (dukesphere) [inline modules/ROOT/pages/tut-af-design-am-flights-api.adoc:125]:
```suggestion
  post: 
```

Comment (dukesphere) [reply modules/ROOT/pages/tut-af-design-am-flights-api.adoc:125]: For the RAML examlpe, `post:` should go right under `get:` with no indentation. See the example in https://dev-docs-internal.kqa.msap.io/beta-code-builder/anypoint-code-builder/tut-af-design-am-flights-api

File: modules/ROOT/pages/tut-af-design-am-flights-api.adoc (hunk 10 above)
Comment (dukesphere) [inline modules/ROOT/pages/tut-af-design-am-flights-api.adoc:114]: See message below about indentation here. Not sure if it applies to OAS, too. I would assume so, but need to check.

File: modules/ROOT/pages/tut-af-design-am-flights-api.adoc [hunk 19]
```diff
@@ -107,9 +145,40 @@ image::get-post-methods-api-console.png["GET and POST methods highlighted in *AP
...
+          required: false
+          schema:
+            type: string
+            enum:
+              - SFO
+              - LAX
+              - CLE
+    post:
+      parameters: []
+
```
Comment (dukesphere) [inline modules/ROOT/pages/tut-af-design-am-flights-api.adoc:174]: Just a note that the number of spaces before `/{ID}:` is different in OAS than RAML. The endpoint shows up in OAS only if `/{ID}:` is aligned with `get`.

File: modules/ROOT/pages/tut-af-design-am-flights-api.adoc (hunk 11 above)
Comment (dukesphere) [reply modules/ROOT/pages/tut-af-design-am-flights-api.adoc:125]: Also note that for OAS, they'll see swiggly lines, so we should probably tell them to _ignore_ errors for now. They'll get `Responses cannot be empty` on the methods (get, post) and another error on the `/{ID}` saying `Endpoint in the API holding a number of executable operations` (TBD if that is an issue)

File: modules/ROOT/pages/tut-af-design-api-add-request-response-details.adoc [hunk 20]
```diff
@@ -39,7 +39,7 @@ Training: American Flight Data Type
 . Select the asset from the *Assets From Exchange* menu.
 . Select the latest version of the asset, such as *v1.0.1*.
 +
-The IDE adds the fragment as a dependency. When the process is complete, check for the `AmericanFlightDataType.raml` asset *Project Dependencies* area of the Explorer. For example:
```
Comment (dukesphere) [inline modules/ROOT/pages/tut-af-design-api-add-request-response-details.adoc:42]: Will need to mention changes to `exchange.json` in the project where the assets show up as dependencies:

```
{
  "dependencies": [
    {
      "version": "1.0.1",
      "classifier": "raml-fragment",
      "assetId": "training-american-flight-data-type",
      "groupId": "68ef9520-24e9-4cf2-b2f5-620025690913",
      "packaging": "zip",
      "provided": false
    },
    {
      "version": "1.0.1",
      "classifier": "raml-fragment",
      "assetId": "training-american-flights-example",
      "groupId": "68ef9520-24e9-4cf2-b2f5-620025690913",
      "packaging": "zip",
      "provided": false
    }
  ],
  "version": "1.0.0",
  "descriptorVersion": "1.0.0",
  "classifier": "oas",
  "main": "american-flights-api-oas-100923.yaml",
  "assetId": "american-flights-api-oas-100923",
  "groupId": "346d7416-8f46-4256-8e45-c85d10787232",
  "name": "American Flights API - OAS 100923"
}
```

Comment (dukesphere) [reply modules/ROOT/pages/tut-af-design-api-add-request-response-details.adoc:42]: It's mentioned in step # 9, but I wonder if we need an example. TBD

File: modules/ROOT/pages/tut-af-design-api-add-request-response-details.adoc [hunk 21]
```diff
@@ -64,20 +64,66 @@ The IDE adds the fragment as a dependency to *Project Dependencies* and  `exchan
...
+* The `AmericanFlight` data type is defined in `AmericanFlightDataType`. 
+* The `AmericanFlightsExample` example is for a `200` response to the `get` request.
 
 To include the fragments: 
 
-. Open *american-flight-api.raml* from the project directory in Explorer.
+. Open *american-flight-api* from the project directory in Explorer.
 . Replace the existing content with this:
-+
+
```
Comment (dukesphere) [inline modules/ROOT/pages/tut-af-design-api-add-request-response-details.adoc:81]: The tabs didn't show up in my build. Do you see them in yours?

File: modules/ROOT/pages/tut-af-design-api-add-request-response-details.adoc [hunk 22]
```diff
@@ -64,20 +64,66 @@ The IDE adds the fragment as a dependency to *Project Dependencies* and  `exchan
...
+====
+OAS Example::
+[source,YAML]
+--
+  openapi: 3.0.0
+   info:
+   title: American Flights API
+
+
+
```
Comment (dukesphere) [inline modules/ROOT/pages/tut-af-design-api-add-request-response-details.adoc:92]: OAS is missing `paths:` here.

File: modules/ROOT/pages/tut-af-design-am-flights-api.adoc [hunk 23]
```diff
@@ -67,27 +67,65 @@ To open the panel:
...
+====
+OAS Example::
+
+[source,YAML]
+--
+openapi: 3.0.0
+info:
+  title: American Flights API
+
+paths:
```
Comment (dukesphere) [inline modules/ROOT/pages/tut-af-design-am-flights-api.adoc:83]: why is `{ID}` listed first under `paths:`? 
To match the RAML example, shouldn't it be just `get:` right now, without `{ID}`?

File: modules/ROOT/pages/tut-af-design-am-flights-api.adoc [hunk 24]
```diff
@@ -67,27 +67,65 @@ To open the panel:
...
+  '/{ID}':
```
Comment (dukesphere) [inline modules/ROOT/pages/tut-af-design-am-flights-api.adoc:112]: same question about `'/{ID}':` here not matching RAML example

File: modules/ROOT/pages/tut-af-design-api-add-request-response-details.adoc [hunk 25]
```diff
@@ -148,9 +221,59 @@ value:
...
+OAS Example::
+[source,YAML]
+--
+ openapi: 3.0.0
+   info:
+   title: American Flights API
+
+
+
+
```
Comment (dukesphere) [inline modules/ROOT/pages/tut-af-design-api-add-request-response-details.adoc:239]: missing `paths:` here in OAS

File: modules/ROOT/pages/tut-af-design-api-add-request-response-details.adoc [hunk 26]
```diff
@@ -148,9 +221,59 @@ value:
...
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: AmericanFlight[]
+
+	post:
+
+'/{ID}':
+ get:
```
Comment (dukesphere) [inline modules/ROOT/pages/tut-af-design-api-add-request-response-details.adoc:263]:
```suggestion
'/{ID}':
  get:
```

Comment (dukesphere) [reply modules/ROOT/pages/tut-af-design-api-add-request-response-details.adoc:263]: formatting here was with a single space, so it didn't align with other entries at this level.

File: modules/ROOT/pages/tut-af-design-api-add-request-response-details.adoc [hunk 27]
```diff
@@ -103,21 +149,24 @@ types: //<1>
...
 +
 image::tut-api-include-directive.png["Include directive in RAML drop-down menu"]
-... Follow the same process to add the example, selecting `exchange_modules` directory and progressing step-by-step to `AmericanFlightsDataType.raml`. For example:
+... Follow the same process to add the example, selecting `exchange_modules` directory and progressing step-by-step to `AmericanFlightsDataType`. For example:
 +
 image::tut-api-include-path.png["Included path in RAML drop-down menu"]
 .. Sets the format of the body of the response to JSON (`application/json`) and configures the type of the response as an array of `AmericanFlight` objects (`AmericanFlight[]`)
-.. Adds the `AmericanFlightsExample.raml` example
+.. Adds the `AmericanFlightsExample` example
 . Proceed to <<post-response-example>>.
```
Comment (dukesphere) [inline modules/ROOT/pages/tut-af-design-api-add-request-response-details.adoc:170]: For `Proceed to <<post-response-example>>.`, need to fix numbering.
The numbering restart at 1.

File: modules/ROOT/pages/tut-af-design-api-add-request-response-details.adoc [hunk 28]
```diff
@@ -128,12 +177,36 @@ Create an example for the post response, and include it in the specification.
 . In the Explorer, right-click on an empty space, and create a folder named `examples`.
 +
 image::add-new-folder.png[alt="New Folder option highlighted in the context menu"]
-. Right-click your `examples` folder, and create a new file named `AmericanFlightExample.raml`.
+. Right-click your `examples` folder, and create a new file named `AmericanFlightExample`.
 +
 image::create-american-fligh-example-file.png[alt="AmericanFlightExample.raml file highlighted"]
 . In the open file, add this example:
 +
-[source,raml]
```
Comment (dukesphere) [inline modules/ROOT/pages/tut-af-design-api-add-request-response-details.adoc:136]: tabs aren't working anywhere in my build

File: modules/ROOT/pages/tut-af-design-api-add-request-response-details.adoc [hunk 29]
```diff
@@ -103,21 +149,24 @@ types: //<1>
...
 [calloutlist]
-.. Uses `!include` to add the `AmericanFlight` object from `AmericanFlightDataType.raml` as a data type
+.. Uses `!include` for RAML files or `$ref` for OAS 3.0 files to add the `AmericanFlight` object from `AmericanFlightDataType.raml` as a data type
 +
-Note that you can use built-in functionality to insert the `!include` directive and file path from the specification file. For example:
+Note that you can use built-in functionality to insert the `!include` or `$ref` directives and file path from the specification file. For example:
 +
 [loweralpha]
-... After typing `AmericanFlight:&nbsp;`(with a space at the end), click Ctrl+Spacebar, start typing `!include`, and select `!include` from the drop-down menu in the RAML file. For example:
+... After typing `AmericanFlight:&nbsp;`(with a space at the end), click Ctrl+Spacebar, start typing `!include` or `$ref`, and select it from the drop-down menu in the RAML file. For example:
```
Comment (dukesphere) [inline modules/ROOT/pages/tut-af-design-api-add-request-response-details.adoc:162]: with the `$ref` they also have to add that hash. See, for example, https://dev-docs-internal.kqa.msap.io/beta-code-builder/anypoint-code-builder/des-create-api-fragments#create-add-fragment-directly

File: modules/ROOT/pages/tut-af-design-api-add-request-response-details.adoc (hunk 28 above)
Comment (dukesphere) [reply modules/ROOT/pages/tut-af-design-api-add-request-response-details.adoc:136]: For a working example, see https://dev-docs-internal.kqa.msap.io/beta-code-builder/anypoint-code-builder/des-create-api-fragments#add-fragments-to-spec (the corresponding adoc file would use the format needed)

File: modules/ROOT/pages/tut-af-design-api-add-request-response-details.adoc [hunk 30]
```diff
@@ -254,13 +455,86 @@ value:
...
+          content:
+            application/json: //<4>
+              schema:
+		examples:
+			message: Flight added (but not really)
+
+
+
+'/{ID}':
+ get:
```
Comment (dukesphere) [inline modules/ROOT/pages/tut-af-design-api-add-request-response-details.adoc:519]:
```suggestion
'/{ID}':
 get:
```
likely spacing issues here, as mentioned before. Doesn't align with other elements at that same level.

File: modules/ROOT/pages/tut-af-design-api-add-request-response-details.adoc [hunk 31]
```diff
@@ -64,20 +64,66 @@ The IDE adds the fragment as a dependency to *Project Dependencies* and  `exchan
...
+            application/json: //<2>
+              schema:
+                type: AmericanFlight[]
+              Examples: //<3>
+                  $ref: '#/components/examples/AmericanFlightsExample'
+
+	post:
+
+'/{ID}':
+ get:
```
Comment (dukesphere) [inline modules/ROOT/pages/tut-af-design-api-add-request-response-details.adoc:119]:
```suggestion
'/{ID}':
 get:
```
Is only one space before `get:` correct here? Doesn't align with other elements at that level.

File: modules/ROOT/pages/tut-af-design-api-add-request-response-details.adoc (hunk 30 above)
Comment (dukesphere) [reply modules/ROOT/pages/tut-af-design-api-add-request-response-details.adoc:519]: @lbauret What should the spacing be here? I didn't suggest a change bc I'm not sure what it's supposed to be.

File: modules/ROOT/pages/tut-af-design-api-add-request-response-details.adoc (hunk 26 above)
Comment (dukesphere) [reply modules/ROOT/pages/tut-af-design-api-add-request-response-details.adoc:263]: @lbauret Is that spacing correct in OAS? We should fix if necessary and add the same formatting for this everywhere in the docs, @failup . Thx!

File: modules/ROOT/pages/tut-af-design-api-add-request-response-details.adoc [hunk 32]
```diff
@@ -128,12 +177,36 @@ Create an example for the post response, and include it in the specification.
 . In the Explorer, right-click on an empty space, and create a folder named `examples`.
 +
 image::add-new-folder.png[alt="New Folder option highlighted in the context menu"]
-. Right-click your `examples` folder, and create a new file named `AmericanFlightExample.raml`.
+. Right-click your `examples` folder, and create a new file named `AmericanFlightExample`.
 +
```
Comment (dukesphere) [inline modules/ROOT/pages/tut-af-design-api-add-request-response-details.adoc:181]: The image shows AmericanFlightsExample.raml, so it doesn't match the description above. Do we need a new image, or can we just clarify somehow? 

How can we fix this?

File: modules/ROOT/pages/tut-af-design-api-add-request-response-details.adoc [hunk 33]
```diff
@@ -254,13 +455,86 @@ value:
...
+			examples: //<3>
+		output: $ref:'#/components/examples/AmericanFlightExample'
+
+
+	post:
+  requestBody:
+   	 content:
+      application/json:
+        schema:
+type: AmericanFlight[]
```
Comment (dukesphere) [inline modules/ROOT/pages/tut-af-design-api-add-request-response-details.adoc:505]: Indentation is incorrect for a lot of this:

```suggestion
  requestBody:
   	 content:
      application/json:
        schema:
type: AmericanFlight[]
```

File: modules/ROOT/pages/tut-af-design-api-add-request-response-details.adoc [hunk 34]
```diff
@@ -254,13 +455,86 @@ value:
...
+            enum:
+              - SFO
+              - LAX
+              - CLE
+      responses://<2>
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
```
Comment (dukesphere) [inline modules/ROOT/pages/tut-af-design-api-add-request-response-details.adoc:494]: The indentation is incorrect for  type: AmericanFlight[]
here and below.

File: modules/ROOT/pages/tut-af-design-api-add-request-response-details.adoc [hunk 35]
```diff
@@ -254,13 +455,86 @@ value:
...
+ get:
+	responses: 
+	'200':
+          description: OK
+          body: 
+            application/json:
+              schema:
+                type: AmericanFlight[]
+			examples: //<5>
+		output: $ref:'#/components/examples/AmericanFlightExample'
```
Comment (dukesphere) [inline modules/ROOT/pages/tut-af-design-api-add-request-response-details.adoc:528]: indentation issue:
		output: $ref:'#/components/examples/AmericanFlightExample'

**Comment by failup** [comment]
Closed because this isnt needed anymore.

**Pull request closed without merging**